src/zap/zapimage.cpp

```diff
@@ -3634,6 +3653,16 @@ ZapNode * ZapImage::GetHelperThunk(CorInfoHelpFunc ftnNum)
     return pHelperThunk;
 }
 
+ZapNode * ZapImage::GetIndirectHelperThunkIfExists(CorInfoHelpFunc ftnNum)
```

Comment:
Is this actually used from anywhere? I can't find it...

Reply:
I somehow undid a change I had made - the newest commit uses this function.